cmd/controller/main.go

```diff
@@ -63,6 +73,7 @@ var (
 	rebaseImage            = flag.String("rebase-image", os.Getenv("REBASE_IMAGE"), "The image used to perform rebases")
 	completionImage        = flag.String("completion-image", os.Getenv("COMPLETION_IMAGE"), "The image used to finish a build")
 	completionWindowsImage = flag.String("completion-windows-image", os.Getenv("COMPLETION_WINDOWS_IMAGE"), "The image used to finish a build on windows")
+	enablePriorityClasses  = flag.Bool("enable-priority-classes", getEnvBool("ENABLE_PRIORITY_CLASSES", false), "if set to true, enables different pod priority classes for normal builds and automated builds")
```

Comment:
This is the first user provided configuration for kpack via a flag. The other flags are not really designed to be modified by the end user. In the past we have discussed a kpack-config config map that would allow users to modify the behavior of kpack (defaults, etc). Would this be a good impetus to do this now?

Reply:
do we definitely need the feature flag? maybe if it's requested?